pkgs/tools/X11/xdg-utils/default.nix

```diff
@@ -1,7 +1,9 @@
 { lib, stdenv, fetchFromGitLab, fetchFromGitHub
 , file, libxslt, docbook_xml_dtd_412, docbook_xsl, xmlto
 , w3m, gnugrep, gnused, coreutils, xset, perlPackages
-, mimiSupport ? false, gawk }:
+, mimiSupport ? false, gawk
+, glib
+, withPortalPatch ? true }:
```

Comment:
Should we stick to upstream by default and only patch when `xdgOpenUsePortal`?

Reply:
Too many rebuilds to turn it on, which makes the setting annoying to use.

Since it's controlled by env should be safe to include the patch always to avoid rebuilds.

Reply:
Oh, I thought we had a `programs.xdg-open` or something where you could simply configure the package but we don't.